### ISSUE TITLE
Image Block: Lightbox - Display image original size when lightbox is open for custom size image.

### ISSUE THREAD

**Issue by jayclydeTags** [WordPress/gutenberg#65247]
## What problem does this address?
When you use a custom-sized image in a lightbox, the lightbox currently shows the custom size. This can be problematic, especially if the original image is a different size. This often occurs in various scenarios, such as:

**E-commerce Websites:**

- Product images in different sizes (thumbnails, product pages, hero images)
- Customer-uploaded images (profile pictures, product reviews)

**Blogs and Content Websites:**
- Featured images in various aspect ratios
- Image galleries with varying image sizes

**Portfolio Websites:**
- Showcase images of different projects
- Client-provided images in various dimensions

## What is your proposed solution?
For custom-sized images, the lightbox should show the original size of the image when opened. This way, users can see the full image without any distortion.

| Size  | Lightbox Open |
| ------------- | ------------- |
| ![image](https://github.com/user-attachments/assets/a882539a-fada-4385-857e-8484ce4e7a59) | ![image](https://github.com/user-attachments/assets/353bd8ef-8704-4277-84af-9071b1761bff)  |
| ![image](https://github.com/user-attachments/assets/d7b30179-ef51-4d3b-b253-14078753daf4)  | ![image](https://github.com/user-attachments/assets/72642dbc-7538-438e-8b34-da42f8efdc73) |

Example:
https://dimsemenov.com/plugins/magnific-popup/
https://www.cssscript.com/demo/lightbox-gallery-image-showcase/
https://www.cssscript.com/demo/image-gallery-popup-havlightbox/

**Comment by michaelbourne** [comment]
A toggle to choose this on a per image basis would be excellent. Many scenarios where we would want the original aspect ratio to show, but not _always_.

**Comment by frspp** [comment]
agree. Would like to have gallery thumbnails as square and lightbox-overlay in orginal aspect ratio.

**Comment by andreawetzel** [comment]
The ability to click on a cropped image and see the image in its original aspect ratio would be a helpful feature for our photography heavy images and galleries as well.
